#23 content strategist stripe vs paypal for squarespace

langrh opened the pull request.

langrh committed:
#23 Content Strategist: Stripe vs PayPal for Squarespace

Create .gitignore file

langrh committed:
#23 Content Strategist: Stripe vs PayPal for Squarespace

.gitignore needed correction

langrh committed:
Uploaded markdown file

justaskdavidb reviewed (changes requested):
Please consider the notes provided in discord about stripe vs PayPal. 
Make sure to watch videos where they are being compared and help the reader make an informed decision on which product is best for them. The similarities in the table you provided are just not divergent enough for someone to "oh I'm going with stripe because I am not selling goods/services" or "oh I'm going with PayPal because its well trusted around the world" or "I'm going with stripe because I want to integrate with Zapier and give a unique User Experience (UX)".

langrh committed:
Updated stripevpaypal.md

Added pros and cons for paypal and stripe on squarespace.

langrh committed:
Update stripevpaypal.md

Fixed error where pros weren't showing for stripe

justaskdavidb committed:
Merge branch 'main' into 23-content-strategist-stripe-vs-paypal-for-squarespace

justaskdavidb committed:
Update stripevpaypal.md

Updated formatting

justaskdavidb committed:
Update stripevpaypal.md

Added author and last update date

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+3. For both, an account must be linked to use.
+4. To create an account for Stripe, navigate to [Stripe Registration](https://dashboard.stripe.com/register).
+5. To create an account for PayPal, navigate to [PayPal Business Signup](https://www.paypal.com/bizsignup).
+6. Once an account is created, link the account on Squarespace and it will be ready for use.
```
This isn't very detailed instruction. Are there at least prompts for folks to follow?

justaskdavidb replied:
We will have to revisit this is a later version. As we will continually update this and other documents over time

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+5. To create an account for PayPal, navigate to [PayPal Business Signup](https://www.paypal.com/bizsignup).
+6. Once an account is created, link the account on Squarespace and it will be ready for use.
+
+## Stripe and PayPal Payment Comparison on Squarespace
```
Delete "on Squarespace"

jasalise replied:
On Squarespace removed

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+| Aspect                       | PayPal                                                                                                    | Stripe                                                                                     |
+|------------------------------|-----------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| **Ease of Use**              | Squarespace makes integration simple. Users can use existing or set up a new account through Squarespace. | Seamless integration with just a Stripe account.                                           |
+| **Transaction Fees**         | Varies by user's location.                                                                                | Varies by country.                                                                         |
```
For Paypal, does that mean it also varies state to state/city to city? And the only variation for Stripe is based on country?

jasalise replied:
Updated Transaction Fees.

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+| **Ease of Use**              | Squarespace makes integration simple. Users can use existing or set up a new account through Squarespace. | Seamless integration with just a Stripe account.                                           |
+| **Transaction Fees**         | Varies by user's location.                                                                                | Varies by country.                                                                         |
+| **Processing**               | Allows checkout without sharing financial info on the merchant site. Payment occurs on PayPal’s website.  | Payments take place on the merchant site with Stripe, providing a simpler user experience. |
+| **Accepted Payment Methods** | PayPal Balance, Visa, Mastercard, American Express, Discover, PayPal Credit.                              | Major credit and debit cards, Apple Pay, Google Pay.                                       |
```
This means both accept major credit and debit cards, right? If there are similarities between the two, the same language should be used between the two.

jasalise replied:
Updated Accepted Payment Methods and added sources.

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+
+| Aspect                       | PayPal                                                                                                    | Stripe                                                                                     |
+|------------------------------|-----------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
+| **Ease of Use**              | Squarespace makes integration simple. Users can use existing or set up a new account through Squarespace. | Seamless integration with just a Stripe account.                                           |
```
I'm not understanding the difference between these two.

jasalise replied:
Updated Ease of Use

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+### Pros:  
+**Trust and Recognition**  
+PayPal is one of the most recognized online payment systems. Customers often trust PayPal and are familiar with its payment process.
+*Source:* [PayPal's About Us page](https://www.paypal.com/us/webapps/mpp/about)  
```
I don't think the sources are necessary in this article.

jasalise replied:
Removed line 36 source

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+*Source:* [PayPal's About Us page](https://www.paypal.com/us/webapps/mpp/about)  
+
+**Buyer Protection**  
+PayPal offers buyer protection which can make potential customers feel more secure about making purchases.  
```
comma before "which"

jasalise replied:
comma added before which

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+Both Squarespace and Stripe are designed for users who might not be technically savvy. They have made it simple to integrate Stripe as a payment method on a Squarespace website. [Squarespace Help](https://support.squarespace.com/hc/en-us/articles/206540797-Stripe)
+
+**Security**: 
+Stripe is PCI compliant, which means that it adheres to high security standards. When integrated with Squarespace, you can reassure your customers that their payment details are safe. [Stripe's Security](https://stripe.com/docs/security/stripe)
```
Spell out what PCI is and follow with PCI in parentheses

jasalise replied:
Added Payment Card Industry (PCI)

KellynBE commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+Like with many other payment processors, Stripe merchants can face chargebacks. If a customer disputes a transaction, there is a possibility that the funds will be reversed, and the merchant might also incur a chargeback fee. [Stripe Documentation on Disputes](https://stripe.com/docs/disputes)
+
+**Dependency on Third-party Platforms**: 
+If either Squarespace or Stripe faces an outage or technical issue, it could it could prevent the busines from processing transactions. This creates a dependency on both platforms' reliability. 
```
- delete second "it could"
- add second "s" on "business"

jasalise replied:
- Removed second "it could"

- corrected business spelling

KellynBE reviewed (changes requested):
Initial editorial review is complete.

dkbaldwin committed:
#23 Content Strategist: Stripe vs PayPal for Squarespace

* fixed overview section

justaskdavidb commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+
+## Overview
+
+Stripe and PayPal are two payment types on Squarespace, each with their own pros and cons. However, both can be used simultaneously on your website. This guide will explain how to set up and use both payment processors, along with a comparison between them.
```
This has been fixed.

jasalise committed:
#23 content strategist stripe vs paypal for squarespace

Removed 2 and 3

jasalise committed:
Update .gitignore

jasalise commented on stripevpaypal.md:
```diff
@@ -0,0 +1,115 @@
+## How to Set Up
+
+1. On Squarespace, navigate to *Commerce -> Payments*.
+2. Once there, PayPal and Stripe will show up as payment methods.
```
Combined steps 1, 2, and 3.

jasalise committed:
#23 content strategist stripe vs paypal for squarespace

Removed On Squarespace
Removed comma before which
Added PCI Payment Card Industry
Removed second it could
Corrected spelling for business

Jandunn commented on stripevpaypal.md:
```diff
@@ -0,0 +1,116 @@
+*Source:* [PayPal's Cross-border Commerce](https://www.paypal.com/us/brc/article/cross-border-commerce)
+
+**Easy Integration**  
+Squarespace offers seamless integration with PayPal. Even users without technical know-how can easily add PayPal as a payment method.  
```
change know how to experience

jasalise replied:
Change made

Jandunn commented on stripevpaypal.md:
```diff
@@ -0,0 +1,116 @@
+While integrating PayPal with Squarespace is easy, customization options for the checkout experience might be limited compared to other payment processors.
+
+**Potential Redirects**  
+Depending on the setup, customers might be redirected to PayPal's website to complete the purchase, which can sometimes result in a disjointed user experience.
```
End sentence at purchase

jasalise replied:
changes made

jasalise committed:
#23 content strategist stripe vs paypal for squarespace

Changed know how to experience
Replaced comma with a period after purchase

jasalise committed:
#23 content strategist stripe vs paypal for squarespace

Removed line 36 source

jasalise committed:
#23 content strategist stripe vs paypal for squarespace

Changed and to then
Updated Ease of Use
Updated Transaction Fees
Updated Accepted Payment Methods

justaskdavidb commented:
@KellynBE , please confirm if your requested changes have been made. I will work with @jasalise to resolve the code conflict as this is her first time encountering it.

jasalise committed:
#23 content strategist stripe vs paypal for squarespace

reverted to master

jasalise committed:
Merge branch 'main' into 23-content-strategist-stripe-vs-paypal-for-squarespace

dkbaldwin committed:
Merge branch 'main' into 23-content-strategist-stripe-vs-paypal-for-squarespace

dkbaldwin reviewed (approved):
Published to Medium and is satisfactory

justaskdavidb reviewed (approved):
LGTM

dkbaldwin committed:
Merge branch 'main' into 23-content-strategist-stripe-vs-paypal-for-squarespace
